jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -129,6 +130,13 @@ public void buildDocker() throws InvalidImageReferenceException, IOException {
       buildConfigurationBuilder.setBaseImageLayersCacheConfiguration(
           applicationLayersCacheConfiguration);
     }
+    if (jibExtension.getUseCurrentTimestamp()) {
+      gradleBuildLogger.warn(
+          "Setting image creation time with current timestamp; your image may not be "
```

Comment:
wording suggestion: "Setting current time as image creation time; ..." or "Setting image creation time to current time; ..."